index.md

```diff
@@ -130,6 +138,11 @@ scoop install pandoc pandoc-crossref
 pip install pandoc-acro pandoc-include --user
 ```
 
+[SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) können mit GNOME `librsvg` Eingebettet werden. Windows muss diese Library Nachtbeziehen.
+```pwsh
+scoop install https://gist.githubusercontent.com/LuminousPath/55c9a416ab8530cd5875e29e8197f22c/raw/fa5ef794486acfa3d130028b9626d24c7d1ed5ad/rsvg-convert.json
+```
```

Comment:
SVG support isn't really in scope for this pr.
I'm also not really a fan of installing scoop manifests directly, as there won't be support for auto-updates, afaik.

Some buckets with `librsvg` can be found [here](https://scoop.sh/#/apps?q=rsvg&s=2&d=1&o=false).

Also, there are a bunch of grammar/spelling mistakes:

````suggestion
[SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) können mit GNOME `librsvg` eingebettet werden. In Windows muss diese Library manuell nachbezogen werden.
```pwsh
scoop install https://gist.githubusercontent.com/LuminousPath/55c9a416ab8530cd5875e29e8197f22c/raw/fa5ef794486acfa3d130028b9626d24c7d1ed5ad/rsvg-convert.json
```
````